packages/broker/src/plugins/brubeckMiner/BrubeckMinerPlugin.ts

```diff
@@ -65,7 +64,11 @@ export class BrubeckMinerPlugin extends Plugin<BrubeckMinerPluginConfig> {
 
         await this.subscribe()
 
-        this.rewardSubscriptionRetryRef = setTimeout(() => this.subscriptionIntervalFn(), this.subscriptionRetryInterval)
+        setAbortableTimeout(
```

Comment:
It would maybe make sense to convert this to use `scheduleAtInterval`. We set a timeout here to execute a function for the first time, and then on the function we set another timeout so that the operation waits `subscriptionRetryInterval` between each invocation.